src/main.cpp

```diff
@@ -189,25 +195,22 @@ namespace rhost {
             return;
         }
 
-#ifdef _WIN32
         static const char mro_banner[] = "Check out Microsoft's enhanced R distribution at https://aka.ms/mrclient. \n\n";
```

Comment:
This technically should be localizable and hence should be in the client. Host can send notification to the client to drop banner to REPL.